cmd/sealer/cmd/image/save.go

```diff
@@ -42,31 +49,46 @@ func NewSaveCmd() *cobra.Command {
 		Long:    longNewSaveCmdDescription,
 		Example: exampleForSaveCmd,
 		Args:    cobra.ExactArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			engine, err := imageengine.NewImageEngine(options.EngineGlobalConfigurations{})
-			if err != nil {
-				return err
-			}
-
-			saveOpts.ImageNameOrID = args[0]
-
-			err = engine.Save(saveOpts)
-			if err == nil {
-				logrus.Infof("successfully save %s to %s", args[0], saveOpts.Output)
-			}
-			return err
-		},
+		RunE:    runSaveCmd,
 	}
 	saveOpts = &options.SaveOptions{}
 	flags := saveCmd.Flags()
-	flags.StringVar(&saveOpts.Format, "format", buildah.OCIArchive, "Save image to oci-archive, oci-dir (directory with oci manifest type), docker-archive, docker-dir (directory with v2s2 manifest type)")
-	flags.StringVarP(&saveOpts.Output, "output", "o", "", "Write image to a specified file")
+
+	formatFlagName := "format"
+	flags.StringVar(&saveOpts.Format, formatFlagName, buildah.OCIArchive, "Save image to oci-archive, oci-dir (directory with oci manifest type), docker-archive, docker-dir (directory with v2s2 manifest type)")
+
+	outputFlagName := "output"
+	flags.StringVarP(&saveOpts.Output, outputFlagName, "o", "", "Write to a specified file (default: stdout, which must be redirected)")
+
+	// TODO: Waiting for implementation, not yet supported
+	flags.StringVar(&loadOpts.TmpDir, "tmp-dir", "", "Set temporary directory when load image. if not set, use system temporary directory(`/var/tmp/`)")
+
 	flags.BoolVarP(&saveOpts.Quiet, "quiet", "q", false, "Suppress the output")
-	flags.BoolVar(&saveOpts.Compress, "compress", false, "Compress tarball image layers when saving to a directory using the 'dir' transport. (default is same compression type as source)")
+
+	compressFlagName := "compress"
+	flags.BoolVar(&saveOpts.Compress, compressFlagName, false, "Compress tarball image layers when saving to a directory using the 'dir' transport. (default is same compression type as source)")
+
+	MultiImageArchiveFlagName := "multi-image-archive"
```

Comment:
its good to try adding more flags to make sealer more powerful , while i have some different advise : 

on pr do one thing, could we split it as two PR, one is adding  `tmp-dir` flag , one is adding another ?